data_collection/gazette/spiders/rj/rj_angra_dos_reis.py

```diff
@@ -0,0 +1,48 @@
+import re
+from datetime import date, datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjAngraDosReisSpider(BaseGazetteSpider):
+    name = "rj_angra_dos_reis"
+    TERRITORY_ID = "3300100"
+    allowed_domains = ["angra.rj.gov.br"]
+    start_date = date(2005, 3, 11)
+
+    def start_requests(self):
+        start_year = int(self.start_date.strftime("%Y"))
+        end_year = int(self.end_date.strftime("%Y"))
+
+        for year in range(end_year, (start_year - 1), -1):
```

Comment:
Essa ordem não importa tanto pois as requisições serão processadas de forma assíncrona. Sugiro deixar em ordem ascendente pra ficar uma besteira mais fácil de ler :)
```suggestion
        for year in range(start_year, end_year + 1):
```

Reply:
Sugestão aplicada.